### ISSUE TITLE
ComposedFixture: Add option to start/stop fixtures in parallel

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#1078]
The `ComposedFixture` currently starts and stops the added fixtures one at a time and this can be quite slow when there's a lot of fixtures (e.g. when emulating an entire service cluster).  It would be nice to have an option that lets this happen parallel.

This would combine well with the new `NeonService` [dependencies feature](https://doc.neonkube.com/T_Neon_Service_ServiceDependencies.htm) that allow services to automatically detect when their dependencies are ready.

**Comment by jefflill** [comment]
Implemented for neonLIBRARY-v2.8.0
